website/src/utils/yamlDataUtils/update.js

```diff
@@ -68,6 +68,20 @@ export const updateConn = (path, value, yamlData) => {
     const steps = path.split(".")
     const target = getValueAtPath(data, steps.splice(0, 3))
     const conn = target.key
+    if (value.toLowerCase() === 'solr' || value.toLowerCase() === 'neo4j') { 
```

Comment:
what is i update my connection name with `solr` or `neo4j`, will this also run the code below? 
will it be safer to validate that the path is pointing to the connection type?

Reply:
we update the type based on what the user has selected, so we use the value that we get from the radio buttons to validate because we have not updated the type yet.

Reply:
this is correct. 
if I will update a text field. 
the value of the textField will be inside the variable value?
can insert the text `solr` into that text field? 
Is this thing can happen?

Reply:
no, this only happens when the protocol is changed